projects/ngx-meta/examples/package.json

```diff
@@ -12,7 +12,9 @@
   "type": "module",
   "devDependencies": {
     "@types/node": "20.12.7",
+    "@types/semver": "7.5.8",
     "execa": "8.0.1",
+    "semver": "7.6.0",
```

Comment:
Side-effect of adding a new lib here is that cache will be invalidated for example apps workflow, yay 🎉

Which would be needed in order to create the new lockfile for Angular <v17 example apps (v15 / v16) now that SSR deps have been added